README.rst

```diff
@@ -62,6 +56,7 @@ Dublin Core
 
 Catalogable Dublin Core
 -----------------------
+
```

Comment:
Out of curiosity, did you use a tool/linter to make these changes? 🤔

Reply:
No, manual. Better readability.